### PR TITLE
Add obsolete and internal load commands.

### PR DESCRIPTION
**Pull request by woodruffw**
Adds `SymsegCommand` (`LC_SYMSEG`), `FvmlimCommand` (`LC_LOADFVMLIB`, `LC_IDFVMLIB`), `IdentCommand` (`LC_IDENT`), and `FvmfileCommand` (`LC_FCMFILE`).

`libmacho` doesn't seem to expose a pubic definition for the structure associated with `LC_PREPAGE`, so that one will have to remain unimplemented for now.

### CODE REVIEW

File: lib/macho/load_commands.rb [hunk 1]
```diff
@@ -1008,4 +1008,68 @@ def version_string
...
+    # @api private
+    def initialize(view, cmd, cmdsize, offset, size)
+      super(view, cmd, cmdsize)
+      @offset = offset
+      @size = size
+    end
+  end
+
+  # An obsolete load command containing a free format string table. Each string
+  # is null-terminated and the command is null-aligned to a multiple of 4.
```
Comment (UniqMartin) [inline lib/macho/load_commands.rb]: I think “zero-padded” is clearer than “null-aligned” or am I misunderstanding this? Any reason why you omitted the “Corresponds to `LC_*`.” comment here and in the next two commands?

**Comment by UniqMartin** [comment]
:+1: Feel free to merge once the minor documentation-related comment has been addressed.

File: lib/macho/load_commands.rb (hunk 1 above)
Comment (woodruffw) [inline lib/macho/load_commands.rb]: > I think “zero-padded” is clearer than “null-aligned” or am I misunderstanding this?

I agree, I'll change this.

The lack of the "corresponds" lines was an oversight as well, I'll add them.

File: lib/macho/load_commands.rb [hunk 2]
```diff
@@ -1046,4 +1046,73 @@ def version_string
...
+    # @return [Fixnum] the library's header address
+    attr_reader :header_addr
+
+    FORMAT = "L=5"
+    SIZEOF = 20
+
+    def initialize(view, cmd, cmdsize, name, minor_version, header_addr)
+      super(view, cmd, cmdsize)
+      @name = LCStr.new(self, name)
+      @header_addr = header_addr
```
Comment (UniqMartin) [inline lib/macho/load_commands.rb]: Dang, I noticed this just now, but what happened to `minor_version`?

Comment (woodruffw) [inline lib/macho/load_commands.rb]: Eek. Fixed it in bcfcb0e.

**Pull request merged**